neptune/new/integrations/transformers/__init__.py

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright (c) 2022, Neptune Labs Sp. z o.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+try:
+    # pylint: disable=import-error
+    from transformers.integrations import NeptuneCallback
```

Comment:
I think we've decided to use wildcard when importing integration from external library in order to be more error-proof after big changes in ingration libraries.

Reply:
But this module has all the integrations (W&B, Tensorboard etc.) :wink:. We could use a wildcard in case of specialized modules like Pytorch Lightning etc.